structuredheadings/plugin.js

```diff
@@ -730,31 +729,33 @@
 
           return [CKEDITOR.plugins.structuredheadings.getCurrentBlockFromPath(editor)];
         },
-
-        exec: function (editor, value) {
-          var selection = editor.getSelection();
-          var headings = CKEDITOR.plugins.structuredheadings.getHeadingsInSelection(
-                editor,
-                selection
-              );
+        exec: function (editor, value) { // eslint-disable-line max-statements
+          var headingIterator = CKEDITOR.plugins.structuredheadings.getHeadingIteratorForSelection(
```

Comment:
Maybe `getHeadingIterator` so that we can use `headingIterator` on line 752 (instead of calling the iterator `heading`)?